fish/PKGBUILD

```diff
@@ -17,14 +17,21 @@ source=("https://github.com/fish-shell/fish-shell/releases/download/${pkgver}/${
         msys2.fish
         msystem.fish
         perlbin.fish)
-sha256sums=('5944da1a8893d11b0828a4fd9136ee174549daffb3d0adfdd8917856fe6b4009'
+sha256sums=('b5b4ee1a5269762cbbe993a4bd6507e675e4100ce9bbe84214a5eeb2b19fae89'
             'SKIP'
             '983c3273e0249957ed6c40785e005739da30f31d4f029383f257f9990d38811a'
             '0af3030b337247c3dfbbe2a6bfa65813e8747044dd3ce48113758b0a41192e09'
             '6c1477bc62facee8353da85e8e7c967c5be33cffe2cd068bdffad381fbc17b48'
             'b136a9fa94abf53e302f7a1cc28def03b58dd2326990c5f02ceb4988341a5ac6')
 validpgpkeys=('003837986104878835FA516D7A67D962D88A709A') # David Adam <zanchey@gmail.com>
 
+sha512sums=('fc50ca44fab3f2d942284d4f714150f7ccf1e49c73da36f8d4ae4a33a9b3280f98bed15848839f5d443b4274fd0ff90174bafa6a8e9a4da226dda63d7766a660'
```

Comment:
Please use one checksum algo, this just complicates things.

Reply:
Sounds good, removed sha256sums and replaced with sha512sums only